features/log-analyzer/org.wso2.carbon.analytics.shared.la.common.feature/src/main/resources/gadgets/LogArtifactDeployed/js/main.js

```diff
@@ -88,10 +88,24 @@ function fetch() {
                 client.searchWithAggregates(queryInfo, function (d) {
                     var obj = JSON.parse(d["message"]);
                     if (d["status"] === "success") {
-                        for (var i = 0; i < obj.length; i++) {
-                            receivedData.push([obj[i].values.artifact, obj[i].values.artifactCountSum]);
+                        try{
+                            for (var i = 0; i < obj.length; i++) {
+                                var ignorePattern = new RegExp(gadgetConfig.tableFieldsFilterPatterns.ignoreCase);
+                                var userPattern = new RegExp(gadgetConfig.tableFieldsFilterPatterns.user);
+                                var artifactPattern = new RegExp(gadgetConfig.tableFieldsFilterPatterns.artifact);
+                                var versionPattern = new RegExp(gadgetConfig.tableFieldsFilterPatterns.version);
+                                var artifact = obj[i].values.artifact;
+                                if(artifact != null && !ignorePattern.test(artifact)){
+                                    receivedData.push([userPattern.exec(artifact), artifactPattern.exec(artifact)[1], versionPattern.exec(artifact)[1], obj[i].values.artifactCountSum]);
+                                }
+                            }
+                            drawDeployedArtifactTable();
+                        }catch (error){
+                            error.message = "Internal server error while data indexing.";
+                            error.status = "Failure";
+                            onError(error);
+                            console.log(error);
                         }
```

Comment:
Hi Manaroma,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/126

Thanks
SajithD